test/unit/js/infiniteGrid.test.js

```diff
@@ -90,8 +90,8 @@ test("release event handler after destroy", function(assert) {
 
 	// Then
 	setTimeout(function() {
-		equal(scrollCount, 1, "should exist scroll event");
-		equal(resizeCount, 1, "should exist resize event");
+		ok(scrollCount > 0, "should exist scroll event");
```

Comment:
Why is scrollCount changed to range check (> 0) from value check(==1) although triggered once. Is it relative with IE8?